dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulationParameters.java

```diff
@@ -38,7 +38,7 @@ public interface ConfigLoader<E extends Extension<DynamicSimulationParameters>>
     public static final String VERSION = "1.0";
 
     public static final int DEFAULT_START_TIME = 0;
-    public static final int DEFAULT_STOP_TIME = 1;
+    public static final int DEFAULT_STOP_TIME = 10;
```

Comment:
So, this new value is needed to be able to have a contingency start time between those default start and stop. This highlights an issue: these start and stop times shouldn't be integers but doubles. Besides, dynawo expects float numbers according to the jobs.xsd file. Could we take the opportunity to break this?